### PR TITLE
Add exception to all USG software being public

### PR DESCRIPTION
**Pull request by ayaleloehr**
This page had previously said, "_all_ software created by Federal Government employees as a “government work” is in the public domain."  Code developed for National Security Systems is an exception to this that I believe should be made explicit.

### CODE REVIEW

File: pages/reuse.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ description: ""
 
 ## 4. Government-Wide Code Reuse
 
-Under U.S. copyright law, all software created by Federal Government employees as a “government work” is in the public domain and, accordingly, is not subject to copyright protection in the United States.<sup id="fnr23"><a href="#fn23">23</a></sup> However, software created on behalf of the Government by third parties, such as private sector vendors, is subject to copyright protection. Currently, the majority of software solutions used in the Federal Government are developed by third parties.
+Under U.S. copyright law, all software created by Federal Government employees as a “government work”, expect that developed for National Security Systems, as defined in 44 U.S.C. §3542, is in the public domain and, accordingly, is not subject to copyright protection in the United States.<sup id="fnr23"><a href="#fn23">23</a></sup> However, software created on behalf of the Government by third parties, such as private sector vendors, is subject to copyright protection. Currently, the majority of software solutions used in the Federal Government are developed by third parties.
```
Comment (Scotchester) [inline pages/reuse.md:10]: Lots of commas happening in this first sentence now. Maybe put the note about National Security Systems in parentheses, or offset with dashes?

Also, a link to the relevant section of code would be appreciated.

Comment (ayaleloehr) [inline pages/reuse.md:10]: So many commas. Agreed. I'm going to stay out of the grammar part of this and let someone else make that decision. Just wanted to point out that not all government software is in the public domain, that's all. 

US Code reference: https://www.law.cornell.edu/uscode/text/44/3542

**Comment by jaredjennings** [comment]
@ayaleloehr, I didn't see anything in the reference you provided about copyright. [17 USC s. 105](https://www.law.cornell.edu/uscode/text/17/105) doesn't appear to say anything about NSS. I'm pretty sure the term "public domain" here refers strictly to whether something is copyrighted, not whether it is available to the public.

For example, if I worked for the government and wrote some cool new software to show a classification banner, not only would it not be copyrighted, it also wouldn't contain any secrets, if I wrote it right. Convincing my superiors, the security folks or the public affairs department of this, so that it could be publicly released, might be a challenge - but even if it were deemed classified, the software would not be copyrighted by the U.S. Government, and that is all that the "public domain" is about when one is talking about copyright.

In conclusion, I don't see the need for your exception. Also you should use "except" not "expect."

**Comment by konklone** [comment]
I think what @ayaleloehr is referring to here is from [the draft policy's Scope page](https://sourcecode.cio.gov/Scope/):

> Source code developed for National Security Systems, as defined in 44 U.S.C. §3542, is exempt from the requirements of this policy.

**Comment by jaredjennings** [comment]
@konklone, I see that now; thank you for pointing it out. But the change makes an assertion about copyright law and NSS, not about this policy and NSS.

**Comment by mattbailey0** [comment]
Thank you all for your input - I'm closing this pull request given that the final language has now been released. Please feel free to reopen or create a new issue or pull if you'd like to continue the conversation!
